bedrock/settings/static_media.py

```diff
@@ -623,6 +623,13 @@
         ),
         'output_filename': 'css/home.css',
     },
+    'home-variant': {
+        'source_filenames': (
+            'css/newsletter/moznewsletter-subscribe.less',
+            'css/mozorg/home/home-variant.scss',
+        ),
+        'output_filename': 'css/home-variant.css',
```

Comment:
For consistency, `css/home-variant-bundle.css`? (Same with pre-existing `css/home.css` above...)